### PR TITLE
Luse/tozpatron 204/schedule models

### PR DESCRIPTION
**Pull request by lsielewicz**
backend models for mapping data form its API

### CODE REVIEW

File: Toz.Dotnet/Toz.Dotnet.Models/Schedule.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Toz.Dotnet.Models
+{
+    public class Schedule
+    {
+        [JsonProperty("configs")]
+        public List<Config> Configs { get; set; }
+
+        [JsonProperty("reservations")]
+        public List<Reservartion> Reservations { get; set;}
```
Comment (Evirth) [inline Toz.Dotnet/Toz.Dotnet.Models/Schedule.cs]: List<Reserva**r**tion>

File: Toz.Dotnet/Toz.Dotnet.Models/Reservartion.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+using System;
+using Newtonsoft.Json;
+using Toz.Dotnet.Models.JsonConventers;
+
+namespace Toz.Dotnet.Models
+{
+    public class Reservartion
```
Comment (Evirth) [inline Toz.Dotnet/Toz.Dotnet.Models/Reservartion.cs]: Typo Reserva**r**tion

Comment (lsielewicz) [inline Toz.Dotnet/Toz.Dotnet.Models/Reservartion.cs]: ops

Comment (lsielewicz) [inline Toz.Dotnet/Toz.Dotnet.Models/Reservartion.cs]: fixed in 973ebf6

**Pull request merged**